src/libYARP_dev/src/MapGrid2D.cpp

```diff
@@ -1006,9 +1006,7 @@ bool MapGrid2D::getOccupancyData(XYCell cell, double& occupancy) const
         yError() << "Invalid cell requested " << cell.x << " " << cell.y;
         return false;
     }
-    if (m_map_occupancy.safePixel(cell.x, cell.y)==-1 ||
-        m_map_occupancy.safePixel(cell.x, cell.y)==255
```

Comment:
@randaz81 If I see correctly, the CellData is a PixelMono, which in turns is a typedef of unsigned char, so it cannot be `-1`. Coverity suggests to remove the check because it is never false.
Maybe the purpose of the check was different ... can you double check this change?

Reply:
ok!